### PR TITLE
feat: products section rework

### PR DESCRIPTION
**Pull request by aspencer116**
# Reworking the product section
## note: you can ignore code in the docs/ folder. It is code generated for hosting through GitHub Pages
- Maintain a full width, one-up product card at all widths
- Reformat the product section HTML for a more semantic read
- Change the product section Sass to implement the design changes
- Add hover interaction for large screen product cards
- Change/color correct images and add smaller size options to optimize for small screens

### CODE REVIEW

File: src/scss/_product.scss [hunk 1]
```diff
@@ -17,50 +17,56 @@ $bp-3up: 75em; //1200px
...
+      fill: $green-brand-d1;
+      position: absolute;
+      right: 20%;
+      transition: all 300ms ease-in-out;
+      transition-delay: 100ms;
 
-    @media (min-width: $bp-c) {
-      width: 75%;
+      @media (min-width: $bp-f) {
+        margin-top: 19.875em;
```
Comment (asimpson) [inline src/scss/_product.scss]: would this make more sense as just `top`?

Comment (aspencer116) [reply src/scss/_product.scss]: Yeah, I think so

**Comment by asimpson** [comment]
👍 looking good man!

**Comment by aspencer116** [comment]
@asimpson thanks for reviewing this!

**Pull request merged**